.github/workflows/main.yml

```diff
@@ -59,9 +59,10 @@ jobs:
         - target: x86_64-pc-windows-gnu
           rust: stable
           os: ubuntu-latest
-        # - target: asmjs-unknown-emscripten
-        #   rust: stable
-        #   os: ubuntu-latest
+        - target: wasm32-wasi
+          rust: stable
+          os: ubuntu-latest
+          crt_static: yes
```

Comment:
Is this `crt_static` config necessary?

Reply:
From my experiments with WASI yes it is pretty much required all the time; I can try to disable it and see if things still work though.

Reply:
Update: It was not needed. Pretty much the opposite, my bad!